internal/storage/bucket_handle_test.go

```diff
@@ -185,3 +186,43 @@ func (t *BucketHandleTest) TestStatObjectMethodWithMissingObject() {
 
 	AssertTrue(errors.As(error, &notfound))
 }
+
+func (t *BucketHandleTest) TestCopyObjectMethodWithValidObject() {
+	_, error := t.bucketHandle.CopyObject(context.Background(),
```

Comment:
This error variable collides with builtin "error" interface. Please rename it at all places in this file to err.

Reply:
fixed